src/schemes/fluid/weakly_compressible_sph/system.jl

```diff
@@ -215,7 +215,10 @@ end
     return system.pressure[particle]
 end
 
-@inline system_sound_speed(system::WeaklyCompressibleSPHSystem) = system.state_equation.sound_speed
+@inline system_sound_speed(system::WeaklyCompressibleSPHSystem) = sound_speed(system.state_equation)
+@inline system_sound_speed(system::WeaklyCompressibleSPHSystem, pressure, density) = sound_speed(system.state_equation,
```

Comment:
Also never used.

Reply:
Would be needed if we actually used a method with compressibility.